cypress/support/commands.ts

```diff
@@ -54,13 +54,16 @@ Cypress.Commands.overwrite(
           onBeforeLoad(win) {
             options?.onBeforeLoad?.(win)
 
-            // We want to test from a clean state, so we clear the local storage (which clears redux).
-            win.localStorage.clear()
+            // We want to test from a clean state, so we clear the persisted state.
+            win.indexedDB.deleteDatabase('redux')
```

Comment:
This may be done automatically by cypress - you'd need to check the docs.

Reply:
looks like it doesn't: https://github.com/cypress-io/cypress/issues/1208